api/alarms/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="index.css">
+  <script defer src="index.js"></script>
+</head>
+<body>
+  <section>
+    <h2>Create Alarm</h2>
+    <form class="create-alarm">
+      <div class="create-alarm__label">Name</div>
+      <div class="create-alarm__value">
+        <input type="text" name="alarm-name" value="my-alarm">
+      </div>
+
+      <div class="create-alarm__label">Initial delay *</div>
+      <div class="create-alarm__value">
+        <input type="number" step="0.1" name="time-value" min="0" value=1>
+
+        <select name="time-format">
+          <option id="format-minutes" value="min" selected>minutes</option>
+          <option id="format-ms" value="ms">milliseconds</option>
+        </select>
+      </div>
+
+      <div class="create-alarm__label">Alarm period *</div>
```

Comment:
I know it's called `alarmPeriod` in the config but this could be better described as "Alarm repeat period".

Reply:
That feels a little long. How about "Repetition period"?

Reply:
Yes, that's better.